{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Base Image
+FROM python:3.6.6-alpine3.7
+
+# Best practice: if you build contains several layers you should order them
+# from the less frequently changed to the more frequently changed (caching)
+
+# Install tools required for the project
+# Best practice:
+# 1. do not install unnecessary packages, keep it small.
+# 2. keep an alphabetic order to avoid duplicates
+RUN apk add --no-cache --update \
+    build-base \
+    git \
+    libffi-dev \
+    linux-headers \
+    pcre-dev \
+    postgresql-client \
+    python3-dev
+
+# Run as non root user (best practice)
+ENV USERNAME=syapse
+ENV PROJECT_NAME={{cookiecutter.project_dash_slug}}
+WORKDIR /home/$USERNAME/$PROJECT_NAME
+COPY . .
+RUN addgroup -g 303 -S $USERNAME \
+    && adduser -u 303 -S -G $USERNAME -h /home/$USERNAME $USERNAME
+RUN chown -R $USERNAME:$USERNAME /home/$USERNAME/$PROJECT_NAME
```

Comment:
In the future, another thing to keep in mind: since Docker caches in layers, it's often smart to combine them. In this case, you could just add it to the above chain of `RUN` directives.

Reply:
Good point! I will change this